projects/plugins/social/src/class-note.php

```diff
@@ -196,8 +196,20 @@ public function maybe_flush_rewrite_rules( $force = false ) {
 	 * @param array $post_id The Post ID.
 	 */
 	public function override_empty_title( $title, $post_id ) {
-		if ( get_post_type( $post_id ) === self::JETPACK_SOCIAL_NOTE_CPT ) {
-			return wp_trim_words( get_the_excerpt(), 10 );
+		$post = get_post( $post_id );
+
+		if ( isset( $post ) && get_post_type( $post_id ) === self::JETPACK_SOCIAL_NOTE_CPT ) {
+			$post_date = get_the_date( 'd M Y', $post );
+			$post_time = get_the_time( 'H:i', $post );
```

Comment:
May be it's better to get the date and time format from settings?

```suggestion
			$post_date = get_the_date( '', $post );
			$post_time = get_the_time( '', $post );
```